lib/backpex/live_resource.ex

```diff
@@ -1173,19 +1177,17 @@ defmodule Backpex.LiveResource do
   end
 
   defp open_action_confirm_modal(socket, action, key) do
-    init_change = action.module.init_change(socket.assigns)
+    base_schema = action.module.base_schema(socket.assigns)
+
     changeset_function = &action.module.changeset/3
 
     metadata = Resource.build_changeset_metadata(socket.assigns)
 
-    changeset =
-      init_change
-      |> Ecto.Changeset.change()
-      |> changeset_function.(%{}, metadata)
+    changeset = changeset_function.(base_schema, %{}, metadata)
 
     socket =
       socket
-      |> assign(:item_action_types, init_change)
+      |> assign(:item, base_schema)
```

Comment:
Internally, the `form_component` treats the `base_schema` as the `item` (we use the same `form_component` like in the `edit` and `create` view). That's why I kept the name `item`. From the outside it has nothing to do with the item, which is why I chose the name `base_schema`.